src/apiClient.ts

```diff
@@ -148,6 +148,9 @@ export default function APIClient(
             // We need to check event again, because kitblocking
             // can nullify the event
             if (event) {
+                if (mpInstance._RoktManager.isReady()) {
+                    mpInstance._RoktManager.processEvent(event);
+                }
```

Comment:
Was there a reason to put this inside of the logic for kit blocker? There is a tiny edge case where a customer may want to fire the event for the purposes of suppressing the ad, but also block the event from being sent to a kit.  I'd suggest moving this out of this if block and towards the top of this function